play-json/shared/src/test/scala/play/api/libs/json/MacroSpec.scala

```diff
@@ -73,26 +67,26 @@ class MacroSpec extends AnyWordSpec with Matchers with org.scalatestplus.scalach
     }
 
     "ignore Option alias" in {
-      def a = {
-        implicit lazy val _: Reads[OptionalInt] = ???
+      def a: Reads[UsingAlias] = {
+        implicit lazy val x: Reads[OptionalInt] = ???
```

Comment:
Not used, so I would rather keep `_`

Reply:
Can't in Scala 3.  I can't remember what error is right now...

Reply:
So there will be a Scala2 warning

Reply:
Yeah, I don't like the situation either, but I didn't have any more time/patience to dig into it, sorry.